apps/backoffice-v2/src/lib/react-query/query-client.ts

```diff
@@ -15,6 +15,25 @@ export const isErrorWithCode = (error: unknown): error is IErrorWithCode => {
   return isObject(error) && 'code' in error && typeof error.code === 'number';
 };
 
+export const isErrorWithMessage = (error: unknown): error is IErrorWithMessage => {
+  return isObject(error) && 'message' in error && typeof error.message === 'string';
+};
+
+async function clearAuthenticatedUser(queryClient: QueryClient) {
+  try {
+    await queryClient.cancelQueries();
+
+    queryClient.clear();
+
+    const authenticatedUser = authQueryKeys.authenticatedUser();
+
+    queryClient.setQueryData(authenticatedUser.queryKey, {
```

Comment:
Does this matter after `queryClient.cancelQueries`?

Reply:
its seems its works like that all over the application